pkg/application/app.go

```diff
@@ -98,6 +98,10 @@ func (app *Avalanche) GetLocalDir(clusterName string) string {
 	return filepath.Join(app.baseDir, constants.LocalDir, clusterName)
 }
 
+func (app *Avalanche) GetLogDir() string {
```

Comment:
nit: we should also make a small change to \`func (app *Avalanche) GetAPMLog()` to use this func